documentation/specifications/move_adapter/README.md

```diff
@@ -263,7 +263,7 @@ gas currency. This can fail with a status code of
 `CURRENCY_INFO_DOES_NOT_EXIST` if the exchange rate cannot be retrieved.
 
 * If the transaction payload is a `ScriptFunction`, check if the on-chain
-Diem Version number is 2 or later. For version 1, validation will fail with
+Aptos Version number is 2 or later. For version 1, validation will fail with
 a `FEATURE_UNDER_GATING` status code.
```

Comment:
this line is just wrong since there's no Diem version any more and ScriptFunction is everywhere